src/routes/twilio.js

```diff
@@ -18,6 +18,9 @@ exports.plugin = {
             body: Joi.string().required(),
           }).required(),
         },
+        pre: [
+          { method: authorizeSendSms },
```

Comment:
Est-ce qu'il ne faudrait pas aussi verifier que le `to` (supposément `companyName`) est bien celui de l'entreprise ?